docs/sources/monitor/monitor-windows.md

```diff
@@ -0,0 +1,299 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tutorials/scenarios/monitor-windows/
+description: Learn how to use Grafana Alloy to monitor Windows servers and desktops
+menuTitle: Monitor Windows
+title: Monitor Microsoft Windows servers and desktops with Grafana Alloy
+weight: 250
+---
+
+# Monitor Microsoft Windows servers and desktops with {{% param "FULL_PRODUCT_NAME" %}}
+
+Microsoft Windows provides tools like Performance Monitor and Event Viewer to track system performance metrics and event logs.
+You can use {{< param "PRODUCT_NAME" >}} to collect your performance metrics and event logs, forward them to a Grafana stack, and create a Grafana dashboard to monitor your Windows performance and events.
+
+The `alloy-scenarios` repository provides series of complete working examples of {{< param "PRODUCT_NAME" >}} deployments.
+You can clone the repository and use the example deployments to understand how {{< param "PRODUCT_NAME" >}} can collect, process, and export telemetry signals.
+
+In this example scenario {{< param "PRODUCT_NAME" >}} collects Windows performance metrics and Windows event logs, and forwards them to a Loki destination..
+
+## Before you begin
+
+This example requires:
+
+* Docker
+* Git
+* Windows Server or Desktop. This scenario monitors a computer running Windows.
+* Windows administrator access. You use administrator access to install {{< param "PRODUCT_NAME" >}} and configure it to collect metrics and logs.
+
+{{< admonition type="note" >}}
+The `docker` commands require administrator privileges.
+{{< /admonition >}}
+
+## Clone the repository
+
+Clone the {{< param "PRODUCT_NAME" >}} scenarios repository.
+
+```shell
+git clone https://github.com/grafana/alloy-scenarios.git
+```
+
+## Deploy the Grafana stack
```

Comment:
Why not just send the data to Grafana Cloud? And we can add a link to a doc that explains how to run Grafana locally if you wish.

Reply:
Could definitely provide them an example of how to connect this to a Grafana Cloud account but does add a barrier to entry where these allow users to spin up a docker compose and see immediately what the Alloy config does. I am open to providing cloud setup instructions in repo aswell though